Makefile

```diff
@@ -28,13 +32,13 @@ cluster-build:
 
 # TODO: Add deepcopy generation script/target
 .PHONY: generate
-generate: bindata crd
+generate: sync-ingress-crd bindata crd
 
 .PHONY: bindata
 bindata:
```

Comment:
`bindata` now depends on `sync-ingress-crd`.

Reply:
I refactored with commit ` 71b113b` for the `crd` and `verify-crd` targets to make both internal and vendored api's as suggested by https://github.com/openshift/cluster-ingress-operator/pull/319/files#r341188193.

Reply:
Really, `bindata` probably should depend on `crd`, but we can get that in a follow-up.